requirements/rocm.txt

```diff
@@ -1 +1,2 @@
 # Extra dependencies for AMD ROCm
+instructlab-training[rocm]>=0.5.0
```

Comment:
Just as a heads up, this change will require a new release of the training library (`0.5.3` I believe) once https://github.com/instructlab/training/pull/255 merges

@fabiendupont once this PR is merged to `main` we will need to backport it to `release-v0.19` - I can assist with that

Reply:
We need to wait until https://github.com/instructlab/training/pull/260 has landed and another release of instructlab-training is available.

Reply:
An hour ago, the dependency would have failed. 0.5.3 and earlier do not have a `rocm` extra. 0.5.4 added the extra.

Reply:
No building will occur until `0.19.3.` is released